analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -70,8 +70,19 @@ void Test_Out()
         void Test_Struct()
```

Comment:
While `int?` is a `struct`, doesn't look like one syntactically.
So, I would propose to change the name of the method into `Test_NullableValueTypes`, or something on that line.